java_src/foam/dao/PMDAO.java

```diff
@@ -0,0 +1,23 @@
+package foam.dao;
+
+import foam.core.FObject;
+import foam.nanos.pm.PM;
+
+/**
+ * Created by nick on 19/05/17.
+ */
+public class PMDAO extends ProxyDAO {
+
+  @Override
+  public FObject put(FObject obj) {
+    PM pm = new PM(PMDAO.class, obj.getClassInfo().getId());
```

Comment:
Should append ':put' to the second argument. Should do the same thing for the find(), remove() and removeAll().